src/securemul.rs

```diff
@@ -283,7 +283,7 @@ mod tests {
         Ok(validate_and_reconstruct(result_shares).into())
     }
 
-    fn make_context<'a>(
+    pub(crate) fn make_context<'a>(
```

Comment:
I think that it's OK to make this an unconditional pub with `pub(crate)` on the test module.